### ISSUE TITLE
Add option to build without the sexpp and g10 keyrings support.

### ISSUE THREAD

**Issue by ni4** [rnpgp/rnp#2252]
## Description

Not all library users would require this functionality, so they would have a chance to make their installation smaller and with less dependencies.

**Comment by Apteryks** [comment]
I'd like to be able to build rnp against a library provided sexpp; is this already possible? The package in Guix symlinks its source to src/libsexpp, which suggests it gets built as part of rnp's build.

**Comment by ni4** [comment]
@Apteryks yeah, there is a CMake's define SYSTEM_LIBSEXPP which allows this.
